composer.json

```
@@ -52,13 +52,7 @@
         "tests": [
             "vendor/bin/phpunit --verbose"
         ],
-        "tests-travis": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml"
-        ],
-        "tests-report": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-html test/html-report"
-        ],
-        "phpcs": [
+       "phpcs": [
             "vendor/bin/php-cs-fixer fix --verbose --diff --dry-run"
```

```suggestion
            "vendor/bin/php-cs-fixer fix --verbose --diff"
```

Don't you think it is more convenient to just fix the errors instead of looking at them?

---

It's the workflow that I and others are used to, and the same on all three PPH SDKs. So I do not want to change this.